Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -98,6 +98,16 @@ open class RouteOptions: DirectionsOptions {
             self.arriveBy = arriveBy
         }
     }
+    
+    /**
+     Initializes a route options object for routes between the given waypoints and an optional profile identifier.
+
+     - parameter waypoints: An array of `Waypoint` objects representing locations that the route should visit in chronological order. The array should contain at least two waypoints (the source and destination) and at most 25 waypoints. (Some profiles, such as `ProfileIdentifier.automobileAvoidingTraffic`, [may have lower limits](https://www.mapbox.com/api-documentation/#directions).)
+     - parameter profileIdentifier: A string specifying the primary mode of transportation for the routes. `ProfileIdentifier.automobile` is used by default.
+     */
+    public convenience init(waypoints: [Waypoint], profileIdentifier: ProfileIdentifier? = nil) {
```

Comment:
Let’s add a test that subclasses RouteOptions and overrides just this initializer, not `init(waypoints:profileIdentifier:queryItems:)`. It doesn’t have to do anything; we just need to see if it would compile. I’m concerned that the `required` bit means it still needs to be implemented even if the convenience initializer would call the required one.

Reply:
Is it sufficient to subclass `RouteOptions`, override the convenience init, and initialize a `RouteOptionsSubclass` object through the override?